jobs/test_api_soc.py

```diff
@@ -0,0 +1,15 @@
+from django.test import TestCase
+
+# Create your tests here.
+from django.urls import reverse
+
+
+class SocCodesAPITests(TestCase):
+    def test_soccodes_urls_return_200(self):
+        """
+        Make sure that url returns 200 status code
+        """
+        # True at the end makes it follow.
```

Comment:
What does this comment mean? is it necessary?

Reply:
removed. unnecessary.